StoreQuestionRequest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Contracts\Validation\Validator;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Http\Exceptions\HttpResponseException;
+
+class NewsRequest extends FormRequest
```

Comment:
@javiershaka this name should have been StoreQuestionRequest probably?

Reply:
yeap, only copy one controller in my desktop, change class.